pkgs/development/tools/yamlfmt/default.nix

```
@@ -2,18 +2,18 @@
 
 buildGoModule rec {
   pname = "yamlfmt";
-  version = "0.12.1";
+  version = "0.13.0";
 
   src = fetchFromGitHub {
     owner = "google";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-VAXDkD4JLoiTomCNtuex/ZEAZtWDEEg5cxETYemvQW8=";
+    hash = "sha256-B1bRYG7ldewdyK8K2Yy5liQcEqv/3/67cQD8JKp8vQI=";
   };
 
   vendorHash = "sha256-UfULQw7wAEJjTFp6+ACF5Ki04eFKeUEgmbt1c8pUolA=";
 
-  doCheck = false;
+  ldflags = [ "-s" "-w" ];
```

How about filling version?

```suggestion
  ldflags = [ "-s" "-w" "-X=main.version=${version}" "-X=main.commit=${src.rev}" ];
```


It changes 

```
yamlfmt dev (none)
```

to

```
yamlfmt 0.13.0 (v0.13.0)
```

I don't have confidence. We should fill git ref as pdfcpu or these src.rev.

https://github.com/NixOS/nixpkgs/blob/747d748f3c041e199add6df38555adc176eabfeb/pkgs/applications/graphics/pdfcpu/default.nix#L11-L58

But at least having a major version helps me a lot.

---

> I don't have confidence. We should fill git ref as pdfcpu or these src.rev.

Let's use `src.rev` for now.

---

Thank you!

```console
> nix-build --attr pkgs.yamlfmt.passthru.tests
/nix/store/k8mzr3a7cbzv2md4csib1lkv3zl4yf2k-yamlfmt-0.13.0-test-version
```

✅ added tests are also passed 👍